File: sandbox-runner/patterns/cancellation-propagation/go/shared.go
package main

const (
	TaskQueue        = "cancellation-propagation-task-queue"
	WorkflowIDPrefix = "cancellation-propagation"
)

// Steps is the set of fulfillment steps run concurrently as child workflows.
var Steps = []string{"reserve-inventory", "authorize-payment", "book-shipping"}

File: sandbox-runner/patterns/cancellation-propagation/java/src/main/java/Shared.java
import java.util.List;

public final class Shared {
    public static final String TASK_QUEUE = "cancellation-propagation-task-queue";
    public static final String WORKFLOW_ID_PREFIX = "cancellation-propagation";
    public static final List<String> STEPS =
            List.of("reserve-inventory", "authorize-payment", "book-shipping");

    private Shared() {}
}
